includes/ucf-news-layouts.php

```diff
@@ -0,0 +1,111 @@
+<?php
+/**
+ * The default functions for the header card layout
+ **/
+
+if ( ! function_exists( 'ucf_news_display_header_card_before' ) ) {
+	function ucf_news_display_header_card_before( $content, $items, $args, $display_type ) {
+		ob_start();
+	?>
+		<div class="ucf-news header-card-layout">
+	<?php
+		return ob_get_clean();
+	}
+
+	add_filter( 'ucf_news_display_header_card_before', 'ucf_news_display_header_card_before', 10, 4 );
+}
+
+if ( ! function_exists( 'ucf_news_display_header_card_title' ) ) {
+	function ucf_news_display_header_card_title( $content, $items, $args, $display_type ) {
+		$formatted_title = $args['title'];
+
+		switch( $display_type ) {
+			case 'widget':
+				break;
+			case 'default':
+			default:
+				if ( $formatted_title ) {
+					$formatted_title = '<h2 class="ucf-news-title">' . $formatted_title . '</h2>';
+				}
+				break;
+		}
+
+		return $formatted_title;
+	}
+
+	add_filter( 'ucf_news_display_header_card_title', 'ucf_news_display_header_card_title', 10, 4 );
+}
+
+if ( ! function_exists( 'ucf_news_display_header_card' ) ) {
+	function ucf_news_display_header_card( $content, $items, $args, $display_type, $fallback_message='' ) {
+		if ( $items === false ) {
+			$items = array();
+		}
+		else if ( ! is_array( $items ) ) {
+			$items = array( $items );
+		}
+
+		$per_row = intval( $args['per_row'] );
+		$show_image = filter_var( $args['show_image'], FILTER_VALIDATE_BOOLEAN );
+
+		ob_start();
+
+		echo '<div class="ucf-news-header-card-deck">';
+
+	if ( count( $items ) === 0 ) : echo $fallback_message; else :
+
+		foreach ( $items as $index=>$item ) :
+			$date = date( "M d", strtotime( $item->date ) );
+			$item_img = UCF_News_Common::get_story_img_tag( $item );
```

Comment:
Let's make sure the project readme is updated to move the UCF News Plugin from the Supported Plugins list to the Required Plugins list, now that we are depending on the `UCF_News_Common` class being defined here.